### PR TITLE
Iter 3 - jaxb

### PR DESCRIPTION
**Pull request by jinocencio-solvd**
### Task

Requirements

- Add JAXB annotations to the hierarchy. Date, List, and complex objects should be covered.
    - JAXB annotations are included for each model in the `models` package. `utils/xml/jaxb/DateAdapter.java` adapts `java.sql.Date` and `utils/xml/jaxb/TimeAdapter.java` adapts `java.sql.time` types for JAXB annotations.
- Parse XML using JAXB.
    - `utils/xml/jaxb/JAXBUtil.java` implements `unmarshallOne(File file, Class<?> clazz)` to parse thru an XML file and return appropriate objects. This is demonstrated in `Main` with `marshallOneXmlOut()` and associated test class `src/test/.../utils/xml/jaxb/JAXBUtilTest.java`.

### CODE REVIEW

**Comment by jinocencio-solvd** [comment]
Auto formatting made changes to a lot of my files. The files that are important for this PR are all files in the `models` package, in `src/main/java/com/laba/utils/xml/jaxb`, in associated test package, and in `Main`

File: XML_Export/Person.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (ankochergin) [inline XML_Export/Person.xml]: later we will work with another type of file, rename package to `export`

File: src/main/java/com/laba/utils/xml/jaxb/JAXBUtil.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+public class JAXBUtil {
+
+    private static final Logger LOG = LogManager.getLogger(JAXBUtil.class);
+
+    public static void marshallManyXmlOut(List<?> objList) {
+        Class<?> objClass = objList.get(0).getClass();
+        try {
+            String path = xmlOutputDir + objClass.getSimpleName() + ".xml";
+            OutputStream outPath = new FileOutputStream(path);
+            String parentTagName = objClass.getSimpleName().toLowerCase() + "_parent";
```
Comment (ankochergin) [inline src/main/java/com/laba/utils/xml/jaxb/JAXBUtil.java]: Nice idea to create universal method for all types! 
`someobjecttype_parent` sounds a bit odd for me, maybe someobjecttype_list is better. Do you have any other ideas?

File: src/main/java/com/laba/utils/xml/jaxb/JAXBUtil.java [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+            JAXBContext context = JAXBContext.newInstance(objClass);
+            Marshaller marshaller = context.createMarshaller();
+            marshaller.setProperty(Marshaller.JAXB_FORMATTED_OUTPUT, Boolean.TRUE);
+            marshaller.setProperty(Marshaller.JAXB_FRAGMENT, Boolean.TRUE);
+
+            XMLOutputFactory xof = XMLOutputFactory.newFactory();
+            XMLStreamWriter xsw = new IndentingXMLStreamWriter(
+                xof.createXMLStreamWriter(outPath, "UTF-8"));
```
Comment (ankochergin) [inline src/main/java/com/laba/utils/xml/jaxb/JAXBUtil.java]: get "UTF_8" from StandardCharsets class or some other library? maybe from some apache library

File: src/main/java/com/laba/utils/xml/jaxb/JAXBUtil.java [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+import org.apache.logging.log4j.Logger;
+
+public class JAXBUtil {
+
+    private static final Logger LOG = LogManager.getLogger(JAXBUtil.class);
+
+    public static void marshallManyXmlOut(List<?> objList) {
+        Class<?> objClass = objList.get(0).getClass();
+        try {
+            String path = xmlOutputDir + objClass.getSimpleName() + ".xml";
```
Comment (ankochergin) [inline src/main/java/com/laba/utils/xml/jaxb/JAXBUtil.java]: ".xml" string is used several times here, consider extracting it to some final variable

File: src/main/java/com/laba/utils/xml/jaxb/JAXBUtil.java (hunk 2 above)
Comment (jinocencio-solvd) [reply src/main/java/com/laba/utils/xml/jaxb/JAXBUtil.java]: i agree, renamed using "_container" instead

**Pull request merged**